docs/source/installing-brave.rst

```diff
@@ -14,6 +14,21 @@ Release Channel Installation
 
 .. highlight:: console
 
+Debian
```

Comment:
Do we also want to add a beta/nightly section that looks a lot like this?

Reply:
Are those available in repos?